app/controllers/reports/regions_controller.rb

```diff
@@ -22,6 +22,7 @@ def show
     @quarterly_registrations = @data[:quarterly_registrations]
     @last_registration_value = @data[:cumulative_registrations].values&.last || 0
     @new_registrations = @last_registration_value - @data[:cumulative_registrations].values[-2]
+    @adjusted_registration_date = @data[:adjusted_registrations].keys[-4]
```

Comment:
think you'll need this in the details action as well

Reply:
Thanks!